### ISSUE TITLE
Please sign the binaries

### ISSUE THREAD

**Issue by elharo** [jbosstools/m2e-apt#70]
![image](https://user-images.githubusercontent.com/1005544/48637733-98a63700-e99c-11e8-8c53-e4057ad5e05c.png)

**Comment by fbricon** [comment]
Question for @nickboldt

**Comment by nickboldt** [comment]
I think what Fred is suggesting is that you could build the bits with YOUR key, not ours. 

All you need from our end is to clone this repo and build it in your own Jenkins, perhaps forking the code to enable signing if it's done like Eclipse.org does it with a maven mojo.

**Comment by fbricon** [comment]
moving m2e-apt to the EF is one solution of course, but I dread the red tape involved with the move

**Comment by fbricon** [comment]
actually since m2e-apt is a dependency of jdt.ls, we're probably supposed to sign it from the EF servers anyway. I'll have to look into that

**Comment by elharo** [comment]
If we signed with our own key, we'd then have to push our version to the Eclipse Marketplace; and I really don't want to fork or maintain it. 

Even more importantly, before I signed it, I should be confident in its code. Right now I know nothing about the code, and can't meaningfully vouch for it.

**Comment by rgrunber** [comment]
Yes, all bundles distributed by Orbit at Eclipse would get signed automatically. However it's rare to host Eclipse plugins. Generally Orbit hosts 3rd party libraries (often lacking OSGi metadata), but it might not be the first time for such an approach.

However, @nickboldt  is right. You could easily set up a signing service on your build servers. In the past I've used http://codeandme.blogspot.com/2017/04/host-your-own-eclipse-signing-server.html to set up a basic local signing service for testing certain capabilities that required signing.

**Comment by martinlippert** [comment]
I would also be interested in consuming signed binaries of this project... :-)

**Comment by mickaelistria** [comment]
m2e-apt's code is now included in https://github.com/eclipse-m2e/m2e-core , please consider reporting issue to https://github.com/eclipse-m2e/m2e-core/issues if it's still relevant.
